roles/accprovision/manifests/net-operator-controller-deployment.yaml

```diff
@@ -0,0 +1,464 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: aci-containers-system
```

Comment:
this is okay for now, but over time we should change this to be network-operator-controller